### DISCUSSION TITLE
source .bash_profile for help

### DISCUSSION THREAD

**Discussion by antyang26** [adoptium/adoptium-support#407] [Q&A] [answered]
1. brew install maven
2. success
3. mvn -v
4. the info like this: 
                   The JAVA_HOME environment variable is not defined correctly,
                   this environment variable is needed to run this program.
5. then i change bash_profile ,like this 
   JAVA_HOME=//Library/Java/JavaVirtualMachines/openjdk-11.jdk/Contents/Home
   PATH=$JAVA_HOME/bin:$PATH:.
   CLASSPATH=$JAVA_HOME/lib/tools.jar:$JAVA_HOME/lib/dt.jar:.
    export JAVA_HOME
    export PATH
    export CLASSPATH
6. restart bash_profile : source .bash_profile
\u@\h \[\033[01;36m\]\W\[\033[01;32m\]$(git_branch)\[\033[00m\] \$

7. run ->  mvn -v
8. the info also like this
  The JAVA_HOME environment variable is not defined correctly,
  this environment variable is needed to run this program.

I don't now the meaning ，and what can i do

**Comment by karianna** [comment, marked as answer]
This isn't really an Adoptium Support issue but more of a homebrew one.

It very much looks like you have other issues in your .bash_profile relating to your git setup and/or hidden characters.

**Reply by antyang26** [reply]
thank you, I think you are right,  but I don't know how to solve the problem . 
Any suggestion ， I will be try
